commodore/cli.py

```diff
@@ -61,16 +61,23 @@ def clean(config: Config, verbose):
                     'the Lieutenant API or fetch/push Git repositories.'))
 @click.option('--push', is_flag=True, default=False,
               help='Push catalog to remote repository.')
+@click.option('--git-author-name', envvar='GIT_AUTHOR_NAME', metavar='USERNAME', help='Name of catalog commit author')
```

Comment:
Should we add default values here? I think currently a name & mail is required without any default/fallback.

Reply:
Never mind, the default is the local Git info.